src/fragments/lib/logging/common/change_log_levels/change_log_levels.mdx

```diff
@@ -0,0 +1,66 @@
+## Change Default Log Level
+In this section, you will learn about how to configure the logging level in your application when using the Amplify logger. This is helpful to help you determine the logging level that works best for your use cases, and which levels of errors or warnings you want to capture in Amazon Cloudwatch.
+
+Below is an example of setting the default log level to `WARN`:
+
+import iosDefaultLogLevel from '/src/fragments/lib/logging/ios/change_log_levels/10_default_log_level.mdx';
+import androidDefaultLogLevel from '/src/fragments/lib/logging/android/change_log_levels/10_default_log_level.mdx';
+import commonDefaultLogLevel from '/src/fragments/lib/logging/common/change_log_levels/10_default_log_level.mdx';
+
+<BlockSwitcher>
+    <Block name="With Configuration File">
+        <Fragments fragments={{all: commonDefaultLogLevel}}/>
+    </Block>
+    <Block name="With Code">
+         <Fragments
+            fragments={{
+                ios: iosDefaultLogLevel,
+                android: androidDefaultLogLevel
+            }}/>
+    </Block>
+</BlockSwitcher>
+
+## Configure Log Level by Category
+Each Amplify category can be configure with its own log level.  
+
+Below is an example of setting different log levels for Storage and Auth categories:
+
+import iosCategoryLogLevel from '/src/fragments/lib/logging/ios/change_log_levels/30_category_log_level.mdx';
+import androidCategoryLogLevel from '/src/fragments/lib/logging/android/change_log_levels/30_category_log_level.mdx';
+import commonCategoryLogLevel from '/src/fragments/lib/logging/common/change_log_levels/30_category_log_level.mdx';
+
+<BlockSwitcher>
+    <Block name="With Configuration File">
+        <Fragments fragments={{all: commonCategoryLogLevel}}/>
+    </Block>      
+    <Block name="With Code">
+        <Fragments
+            fragments={{
+                ios: iosCategoryLogLevel,
+                android: androidCategoryLogLevel
+            }}
+        />  
+    </Block>
+</BlockSwitcher>
+
+## Change local storage maximum size
```

Comment:
can we move this section into its own page please on the left nav

Reply:
Moved into a new page called `Change Local Storage`